actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -556,10 +570,27 @@ def current_page?(options)
       end
 
       private
+        def default_noopener(rel)
+          rel ||= 'noopener'
+          return rel if rel.include?('noopener')
+          return rel.split(' ') << 'noopener'
+        end
+
+        def target_blank?(html_options)
+          html_options['target'] && html_options['target'].include?('_blank')
```

Comment:
I think the problem noopener solves is present no matter the value of the `target` attribute. For example, as best I can tell, `target="foo"` causes the problem just like `target="_blank"`. So perhaps the check here should just be for whether the `target` attribute has a non-blank value.

(Side note: Using values other than `_blank` for `target` used to be used for opening the link in a frame with the matching name. I'm not sure if it has any practical use now, but it still works to open a new tab, at least in Chrome.)